openedx_webhooks/lib/edx_repo_tools_data/models.py

```diff
@@ -125,11 +117,18 @@ def agreement_expires_on(self):
         return expires_on
 
     @property
+    def before(self):
+        """
+        Optional[Dict[str, Dict]]: User's `before` data.
+        """
+        return self._data.get('before')
```

Comment:
I thought we would prefer to keep the "before" details internal, so as not to further expose the before/expired confusion.